src/relay/op/vm/vm.cc

```diff
@@ -41,6 +41,9 @@
 namespace tvm {
 namespace relay {
 
+// shape_of
+TVM_REGISTER_NODE_TYPE(ShapeOfAttrs);
```

Comment:
I've not seen this issue before, perhaps a linker bug. Could you leave a comment in case someone tries to move it back?

Reply:
ok